docs/modules/chat.mdx

```diff
@@ -1,9 +1,12 @@
 ---
 title: Chat
 ---
-:::note
+:::warning
 This addon requires you to have a matching version of EssentialsX installed.
-You may also need [Vault](https://github.com/milkbowl/Vault) if you wish to get prefixes or suffixes from external plugins
+:::
+
+:::note
+[Vault](https://github.com/milkbowl/Vault) is required if you wish to retrieve prefixes or suffixes from your permission manager. We recommend using [LuckPerms](https://luckperms.net/) as your permissions plugin.
```

Comment:
I don't particularly like the wording on this. I would change this to 
"Vault is required if you wish to retrieve prefixes or suffixes from a permission manager." 
I don't think it's necessary to mention LuckPerms in this specific message.

Reply:
Agreed and adjusted mentioned note to remove redundant mention of LuckPerms.